### PR TITLE
Make prepended/appended buttons look more button'y

### LINKED ISSUE #308: Make prepended/appended buttons look more button'y
Currently look like:

screen shot 2016-09-07 at 11 02 59

Will potentially look like:

screen shot 2016-09-07 at 11 02 45

### PR DESCRIPTION
**Pull request by langy**
Resolves #308 

screen shot 2016-09-07 at 11 36 28

### CODE REVIEW

File: views/pulsar/v2/helpers/form.html.twig [hunk 1]
```diff
@@ -948,11 +948,19 @@ controls.class   | string | A space separated list of class names
             {% set input_group_classes = input_group_classes ~ ' is-removable' %}
         {% endif %}
 
+        {% if (options.parent.prepend_type is defined and options.parent.prepend_type == 'button') %}
```
Comment (Stanton) [inline views/pulsar/v2/helpers/form.html.twig]: If `prepend_type` is not passed, will it default to old styling?

Comment (langy) [inline views/pulsar/v2/helpers/form.html.twig]: It'll just not print out the classes, or the button, I presume (as it uses the same if statement as on line 965)

Comment (langy) [inline views/pulsar/v2/helpers/form.html.twig]: Ah, I see your point now. Yes, it will.

Comment (Stanton) [inline views/pulsar/v2/helpers/form.html.twig]: L:965 is an else, so it should pass the button as a regular `input-group-addon`

Comment (langy) [inline views/pulsar/v2/helpers/form.html.twig]: You're right. No change in functionality here.

**Comment by jamesjacobs** [comment]
👍

**Pull request merged**